docs/getting-started/new_project_guide.md

```diff
@@ -334,35 +320,26 @@ It is common for several [fuzz targets]({{ site.baseurl }}/reference/glossary/#f
 to reuse the same dictionary if they are fuzzing very similar inputs.
 (example: [expat](https://github.com/google/oss-fuzz/blob/master/projects/expat/parse_fuzzer.options)).
 
-### Custom libFuzzer options for ClusterFuzz
+### Input Size
 
-By default, ClusterFuzz will run your fuzzer without any options. You can specify
-custom options by creating a `my_fuzzer.options` file next to a `my_fuzzer` executable in `$OUT`:
-
-```
-[libfuzzer]
-close_fd_mask = 3
-only_ascii = 1
-```
-
-[List of available options](http://llvm.org/docs/LibFuzzer.html#options). Use of `max_len` is not recommended as other fuzzing engines may not support that option. Instead, if
-you need to strictly enforce the input length limit, add a sanity check to the
-beginning of your fuzz target:
+By default, the fuzzing engine will generate input of any arbitrary length.
+This is often useful to try unexpected use cases that lead to security
```

Comment:
nit: this sentence does not seem strictly necessary. I think we can cut it without losing any clarity to the reader.

Reply:
Well, we don't want them to add this size limit, so this just tells that why we do arbitrary length and tell that it is useful for security vuln finds and unexpected usecases.